common/lib/xmodule/xmodule/video_module/video_module.py

```diff
@@ -182,6 +189,36 @@ def get_transcripts_for_student(self, transcripts):
         sorted_languages = OrderedDict(sorted_languages)
         return track_url, transcript_language, sorted_languages
 
+    @cached_property
+    def prioritize_hls_flag(self):
```

Comment:
Nice! Now this can be renamed to`hls_enabled`, since it has the sole purpose of checking enablement.